### PR TITLE
add `machine424` and `rexagod`  to OWNERS

### PR DESCRIPTION
**Pull request by rexagod**
Also move `bwplotka` and `arajkumar` to emeritus.

cc @jan--f @machine424

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
 reviewers:
-- bwplotka
 - simonpasquier
 - slashpai
 - prashbnair
-- arajkumar
 - sthaha
 - PhilipGough
 - bison
```
Comment (slashpai) [inline OWNERS:9]: you can move Brad to emeritus too

File: OWNERS [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 - raptorsun
 - jan--f
+- rexagod
+- machine424
 
 approvers:
-- bwplotka
 - simonpasquier
 - slashpai
 - prashbnair
```
Comment (slashpai) [inline OWNERS:19]: Prasanth as well

**Review by simonpasquier — COMMENTED** [review]
sthaha and juanrh can be removed. And you can add @marioferh too.

**Comment by simonpasquier** [comment]
We can probably skip this repository since we don't use it for anything :)

**Comment by rexagod** [comment]
I've pushed a commit based on reviews, but feel free to close this out.

**Pull request merged**